internal/tracker/source/tekton/tekton.go

```diff
@@ -553,5 +554,20 @@ func enrichPackageFromAnnotations(p *hub.Package, annotations map[string]string)
 		}
 	}
 
+	// Sign key
+	if v, ok := annotations[signatureAnnotation]; ok {
+		var signature string
+		if err := yaml.Unmarshal([]byte(v), &signature); err != nil {
+			errs = multierror.Append(errs, fmt.Errorf("%w: invalid signature value", errInvalidAnnotation))
+		} else {
+			if signature == "" {
```

Comment:
Can we be a bit more permissive for now and not require the signature to be present? If we return an error when the signature isn't there, the package won't be processed.

Reply:
Ahh, good point! I made some modifications, PTAL.